psalm.xml.dist

```diff
@@ -42,6 +42,11 @@
                 <file name="src/Container/ServerRequestErrorResponseGeneratorFactory.php"/>
             </errorLevel>
         </DeprecatedTrait>
+        <UndefinedClass>
+            <errorLevel type="suppress">
+                <referencedClass name="Zend\*"/>
```

Comment:
Is this needed at all?

Reply:
Decided to roll with it as-is: it's not a big deal.

Reply:
Yes it's needed. Psalm complaints about non existing classes otherwise. Because it's all over the please I thought adding it here is easier than everywhere else.

In a new major version we can remove all this stuff and support for old Zend classes.